rackcity/views/asset_views.py

```diff
@@ -202,23 +203,31 @@ def asset_add(request):
             status=HTTPStatus.BAD_REQUEST,
         )
 
-    if (
-        serializer.validated_data["model"].is_rackmount()
-        and "rack" in serializer.validated_data
-        and "rack_position" in serializer.validated_data
```

Comment:
The form should never let this happen

Reply:
so data will always have a field "rack" but it can sometimes be null if the user does not select a rack -> hence leading to us not catching that error. im tracking that bug fix though

Reply:
wait actually, if you want to take over that bug? because I imagine we would want to add similar logic for if a user forgets to select a chassis for a blade slot

Reply:
doesnt need to be done in this PR just flagging it here